### DISCUSSION TITLE
Node Self-Tagging Deprecation

### DISCUSSION THREAD

**Discussion by jriddle-sf** [chef/chef#13381] [Q&A]
Can someone from the Chef Product Team confirm this feature is going to be deprecated?

https://github.com/chef/chef/blob/1f4ec9fd3587ae4efdc3b20b273bd64164328063/lib/chef/node.rb#L404

It was mentioned here and want to know if this is still something we cannot rely on as way to communicate status to an external service. Thanks.

https://stackoverflow.com/questions/39174065/chef-node-client-submits-its-own-tag-to-chef-server

**Comment by Stromweld** [comment]
@jriddle-sf there are no current plans to deprecate this feature. This is a widely used feature and if there is any decision in the future to deprecate it, there would be a planned deprecation process with official announcements to the community via normal channels like chef community slack, blog post, discourse, etc... as well as a timeline to give users plenty of time to refactor things to move away from it before it is officially removed.
